frontend/src/lib/Apis.test.ts

```diff
@@ -182,4 +182,16 @@ describe('Apis', () => {
       },
     );
   });
+
+  it('checks if Tensorboard pod is ready', async () => {
```

Comment:
Thanks for also testing this.
Consider adding a second case that verifies when tensorboard pod is not ready, this also works.

Reply:
done